mongodb_migrations/cli.py

```diff
@@ -123,8 +137,18 @@ def _do_rollback(self, to_datetime=None):
                         print(e.message)
                     raise
                 print("Succeed to downgrade version: %s" % self.migrations[migration_datetime])
+                performed_migrations += 1
                 if not self.config.dry_run:
                     self._remove_migration(migration_datetime)
+                if performed_migrations == levels and levels > 0:
```

Comment:
when `performed_migrations` is `1` and `levels` is `3`, the `if` condition will fail, and desired message won't be printed.